cfme/fixtures/v2v.py

```diff
@@ -163,32 +165,28 @@ def get_vm(request, appliance, source_provider, template, datastore="nfs"):
     return vm_obj
 
 
-def _form_data(source_provider, provider):
-    form_data = {
-        "general": {
-            "name": "infra_map_{}".format(fauxfactory.gen_alphanumeric()),
-            "description": "Single Datastore migration of VM from {ds_type1} to"
-            " {ds_type2}".format(ds_type1="nfs", ds_type2="nfs"),
-        },
-        "cluster": {"mappings": [_form_data_mapping("clusters", source_provider, provider)]},
-        "datastore": {
-            "Cluster ({})".format(provider.data.get("clusters")[0]): {
-                "mappings": [
-                    _form_data_mapping("datastores", source_provider, provider, "nfs", "nfs")
-                ]
-            }
-        },
-        "network": {
-            "Cluster ({})".format(provider.data.get("clusters")[0]): {
-                "mappings": [
-                    _form_data_mapping(
-                        "vlans", source_provider, provider, "VM Network", "ovirtmgmt"
-                    )
-                ]
-            }
-        },
+def infra_mapping_default_data(source_provider, provider):
+    """
+    Default data for infrastructure mapping form.
+    It is used in other methods to recursive update the data accoriding
+    to parameters in tests.
+
+    Args:
+        source_provider: Vmware provider
+        provider: Target rhev/OSP provider
+    """
+    infra_mapping_data = {
+        "name": "infra_map_{}".format(fauxfactory.gen_alphanumeric()),
+        "description": "Single Datastore migration of VM from {ds_type1} to {ds_type2}".format(
+            ds_type1="nfs", ds_type2="nfs"
```

Comment:
`nfs` is just the default parametrization value for datastore, and I don't think should be hardcoded in this default. Why not include `request` here and pick from `request.param` instead, so the test controls which datastore types are parametrized instead of the default being hardcoded.

Reply:
@mshriver : This is default data for a mapping. It is not used in tests . The mapping used in tests is derived from this default data by doing recursive updates from request.params.